backend/src/test/java/com/daedan/festabook/schedule/controller/ScheduleControllerTest.java

```diff
@@ -0,0 +1,169 @@
+package com.daedan.festabook.schedule.controller;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+
+import com.daedan.festabook.organization.domain.Organization;
+import com.daedan.festabook.organization.domain.OrganizationFixture;
+import com.daedan.festabook.organization.infrastructure.OrganizationJpaRepository;
+import com.daedan.festabook.schedule.domain.Event;
+import com.daedan.festabook.schedule.domain.EventDate;
+import com.daedan.festabook.schedule.domain.EventDateFixture;
+import com.daedan.festabook.schedule.domain.EventFixture;
+import com.daedan.festabook.schedule.infrastructure.EventDateJpaRepository;
+import com.daedan.festabook.schedule.infrastructure.EventJpaRepository;
+import io.restassured.RestAssured;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class ScheduleControllerTest {
+
+    private static final String ORGANIZATION_HEADER_NAME = "organization";
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private OrganizationJpaRepository organizationJpaRepository;
+
+    @Autowired
+    private EventDateJpaRepository eventDateJpaRepository;
+
+    @Autowired
+    private EventJpaRepository eventJpaRepository;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+    }
+
+    @Nested
+    class getAllEventDateByOrganizationId {
+
+        @Test
+        void 성공_조직_ID_기반() {
+            // given
+            Organization organization = OrganizationFixture.create("우리 학교");
+            Organization otherOrganization = OrganizationFixture.create("다른 학교");
+
+            organizationJpaRepository.saveAll(List.of(organization, otherOrganization));
+
+            eventDateJpaRepository.saveAll(EventDateFixture.createList(4, organization));
```

Comment:
4 라는 정수값 변수로 빼도 좋을 것 같네요
```suggestion
            int excepted = 4;
            eventDateJpaRepository.saveAll(EventDateFixture.createList(excepted, organization));
            
            
```